DataHub/Integration/02_ManageAttributeSets.md

```diff
@@ -0,0 +1,304 @@
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTE SETS > Button Add*
+
+![Create attribute set inheritance](../../Assets/Screenshots/DataHub/Settings/AttributeSets/CreateAttributeSetInheritance.png "[Create attribute set inheritance]")
+
+1. Select an attribute set in the *Inherit/Copy values from* drop-down list.
+
+2. Enable the *Inherit configuration* toggle if you also want to adopt the configuration to the new attribute set.
+
+3. Select the **Inherit** option in the *Inherit/Copy values* drop-down list.
+
+4. Click the [SAVE] button.   
+  The new attribute set has been saved. The *Create attribute set* view changes to the *Edit attribute set* view. The *Attribute set created* pop-up window is displayed.   
+
+  ![Attribute set created](../../Assets/Screenshots/DataHub/Settings/AttributeSets/AttributeSetCreated.png "[Attribute set created]")     
+
+5. Click the ![Back](../../Assets/Icons/Back02.png "[Back]") (Back) button in the upper left corner to return to the list of attribute sets or proceed to [edit the attribute set](#edit-an-attribute-set).
+
+
+
+### Create an attribute set with attribute copying
+
+When you create an attribute set with attribute copying, you copy all attributes from the selected attribute set once.
+After copying, the attribute set behaves like an attribute set without attribute inheritance.
+Further changes in the copied attribute set or not affecting the created attribute set.   
+
+#### Prerequisites
+
+The basic data of the attribute set has been completed, see [Define the attribute set basic data](#define-the-attribute-set-basic-data).
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTE SETS > Button Add*
+
+![Create attribute set inheritance](../../Assets/Screenshots/DataHub/Settings/AttributeSets/CreateAttributeSetInheritance.png "[Create attribute set inheritance]")
+
+1. Select an attribute set in the *Inherit/Copy values from* drop-down list.
+
+2. Enable the *Inherit configuration* toggle if you also want to adopt the configuration to the new attribute set.
+
+3. Select the **Copy once** option in the *Inherit/Copy values* drop-down list.
+
+4. Click the [SAVE] button.   
+  The new attribute set has been saved. The *Create attribute set* view changes to the *Edit attribute set* view. The *Attribute set created* pop-up window is displayed.   
+
+  ![Attribute set created](../../Assets/Screenshots/DataHub/Settings/AttributeSets/AttributeSetCreated.png "[Attribute set created]")     
+
+5. Click the ![Back](../../Assets/Icons/Back02.png "[Back]") (Back) button in the upper left corner to return to the list of attribute sets or proceed to [edit the attribute set](#edit-an-attribute-set).
+
+
+
+## Edit an attribute set
+
+After you have created an attribute set, you can edit it.
+Depending on the selected inheritance type, only a certain number of attributes may be editable.
+However, you can add further attributes to an attribute set or remove unmapped attributes from an attribute set.
+
```

Comment:
Bei Inheritet schaffe ich nicht, Attributes zu editieren oder hinzuzufügen (Meldung: Attribute-Satz geschützt). Ich denke, das ist normal, denn es ist vererbt, aber vielleicht kurze Info dazu hier? Auch wenn ich versuche, Inherit/Copy Values zu editieren, bekomme ich die Meldung: Attribut Set Map existiert nicht: Attribute set map with attribute set id 762 and attribute id 1072 could not be found. Was mache ich falsch?